### PR TITLE
improve e2e experience 

### PR DESCRIPTION
**Pull request by PratikMahajan**
makes sure that e2e works seamlessly on local machine.
also, added flag `-- --nocapture` to print the logs for tests.

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -206,12 +203,9 @@ run-daemons-e2e:
 	just \
 		registry="{{registry}}" repository="{{repository}}" \
 		run-graph-builder-e2e 2>&1 &
-	PG_PID=$!
 
 	just run-policy-engine 2>&1 &
-	PE_PID=$!
 
-	trap "kill $PG_PID $PE_PID" EXIT
 	sleep infinity
```
Comment (wking) [inline Justfile]: It feels strange to me drop drop this kill but still keep the infinite `sleep`.  Don't you want killing the `just run-daemons-e2e` call to clean up the daemons it created?

File: Justfile [hunk 2]
```diff
@@ -79,12 +80,8 @@ run-e2e:
 	#!/usr/bin/env bash
 	set -e
 
-	just \
```
Comment (vrutkovs) [inline Justfile:82]: Not sure what this change does

Comment (PratikMahajan) [reply Justfile:82]: reverted the change

**Review by vrutkovs — APPROVED** [review]
/lgtm
/hold cancel

**Pull request merged**